### PR TITLE
Fix on boarding screen

### LINKED ISSUE #894: The Element logo in both modes is pixelated
The Element logo in both modes is pixelated - can it be exported and then added in a higher resolution?

_Originally posted by @callumu in https://github.com/vector-im/element-x-android/issues/267#issuecomment-1623568147_

### PR DESCRIPTION
**Pull request by bmarty**
Closes #894 

Also change some preview annotations to be able to render resource with night modifier correctly in Android Studio Preview.

Sadly this will rename the file, so it will be more difficult to compare ScreenShots, there here is a comparison:

|Previous Light|New Light|
|-|-|
|image|image|

|Previous Dark|New Dark|
|-|-|
|image|image|

### CODE REVIEW

File: tests/uitests/src/test/snapshots/images/io.element.android.tests.uitests_ScreenshotTest_preview_tests[io.element.android.features.messages.impl.timeline_null_DefaultGroup_TimelineViewDarkPreview_0_null_6,NEXUS_5,1.0,en].png
Comment (csmith) [inline tests/uitests/src/test/snapshots/images/io.element.android.tests.uitests_ScreenshotTest_preview_tests[io.element.android.features.messages.impl.timeline_null_DefaultGroup_TimelineViewDarkPreview_0_null_6,NEXUS_5,1.0,en].png:1]: I guess this needs changing to `@DayNightPreview` as well to pick the right resource?

File: tests/uitests/src/test/snapshots/images/io.element.android.tests.uitests_ScreenshotTest_preview_tests[io.element.android.features.onboarding.impl_null_DefaultGroup_OnBoardingScreenPreview-D-0_0_null_3,NEXUS_5,1.0,en].png
Comment (csmith) [inline tests/uitests/src/test/snapshots/images/io.element.android.tests.uitests_ScreenshotTest_preview_tests[io.element.android.features.onboarding.impl_null_DefaultGroup_OnBoardingScreenPreview-D-0_0_null_3,NEXUS_5,1.0,en].png:1]: That text seems _uncomfortably_ close to the logo, is it right?

**Review by csmith — APPROVED** [review]
LGTM in general, just two questions about the screenshots

File: tests/uitests/src/test/snapshots/images/io.element.android.tests.uitests_ScreenshotTest_preview_tests[io.element.android.features.messages.impl.timeline_null_DefaultGroup_TimelineViewDarkPreview_0_null_6,NEXUS_5,1.0,en].png
Comment (csmith) [reply tests/uitests/src/test/snapshots/images/io.element.android.tests.uitests_ScreenshotTest_preview_tests[io.element.android.features.messages.impl.timeline_null_DefaultGroup_TimelineViewDarkPreview_0_null_6,NEXUS_5,1.0,en].png:1]: Ah, you did that while I was reviewing :grin: :tada:

File: libraries/designsystem/src/main/kotlin/io/element/android/libraries/designsystem/atomic/atoms/ElementLogoAtom.kt [hunk 1]
```diff
@@ -53,85 +53,82 @@ import io.element.android.libraries.theme.ElementTheme
...
+        outerSize = 158.dp,
+        logoSize = 110.dp,
+        cornerRadius = 44.dp,
+        borderWidth = 1.dp,
+        shadowColorDark = Color.Black.copy(alpha = 0.4f),
+        shadowColorLight = Color(0x401B1D22),
+        shadowRadius = 32.dp,
+    )
+
+    object Huge : ElementLogoAtomSize(
```
Comment (jmartinesp) [inline libraries/designsystem/src/main/kotlin/io/element/android/libraries/designsystem/atomic/atoms/ElementLogoAtom.kt:123]: I actually created the `Large` size for this screen, but I forgot to replace it. You can just remove `Huge` and customize the `Large` one.

Comment (bmarty) [reply libraries/designsystem/src/main/kotlin/io/element/android/libraries/designsystem/atomic/atoms/ElementLogoAtom.kt:123]: Ah OK, thanks, I forgot to check for usage :)

File: tests/uitests/src/test/snapshots/images/io.element.android.tests.uitests_ScreenshotTest_preview_tests[io.element.android.features.onboarding.impl_null_DefaultGroup_OnBoardingScreenPreview-D-0_0_null_3,NEXUS_5,1.0,en].png
Comment (bmarty) [reply tests/uitests/src/test/snapshots/images/io.element.android.tests.uitests_ScreenshotTest_preview_tests[io.element.android.features.onboarding.impl_null_DefaultGroup_OnBoardingScreenPreview-D-0_0_null_3,NEXUS_5,1.0,en].png:1]: Yes, that's not ideal, but it's on a small screen, and this configuration (with 3 buttons) cannot happen yet.

**Pull request merged**